tests/tests/unit-tests/routers/OrganizationRouterTest.js

```diff
@@ -0,0 +1,93 @@
+var blueprint = require ('@onehilltech/blueprint')
+  , request   = require ('supertest')
+  , expect    = require ('chai').expect
+  ;
+
+var appPath         = require ('../../../fixtures/appPath');
+var organizations   = require ('../../../fixtures/organizations');
+
+describe ('OrganizationRouter', function () {
+  before (function (done) {
+    blueprint.testing.createApplicationAndStart (appPath, done)
+  });
+
+  after (function (done) {
+    blueprint.app.models.Organization.remove ({}, done);
+  });
+
+  describe ('/organizations', function () {
+
+    var organization = organizations[0];
+    var organizationId;
+
+    describe ('POST', function (done) {
+      it ('should create an organization in the database', function (done) {
+        request (blueprint.app.server.app)
+          .post ('/organizations') // route
+          .send (organization) // data being sent
```

Comment:
The ResourceControllers expect data in the format:
```
{
    "organization": organization
}
```
meaning you need to wrap the organization object as shown above.